src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java

```diff
@@ -648,6 +648,12 @@ private void validateWork(ResourceEdge edge) {
         TERM.getValue(), List.of("Autonomous")
       ), "Autonomous");
     validateTargetAudience(edgeIterator.next());
+    validateEdge(edgeIterator.next(), PredicateDictionary.LANGUAGE, List.of(LANGUAGE_CATEGORY),
```

Comment:
minor
could you refactor this test, to avoid iterator, use finding edges by logic and add try o minimize validation in one assertion

Reply:
I suggest to create a dedicated technical debt ticket for such changes